assets/src/components/detours/detourPanelComponents.tsx

```diff
@@ -64,3 +71,26 @@ export const AffectedRoute = ({
     </div>
   </section>
 )
+
+export const CopyButton = ({ detourText }: { detourText: string }) => (
+  <OverlayTrigger
+    placement="bottom"
+    trigger="click"
+    rootClose
+    rootCloseEvent="mousedown"
+    overlay={
+      <Popover>
+        <Popover.Body>Copied to clipboard!</Popover.Body>
+      </Popover>
+    }
+  >
+    <Button
+      className="c-diversion-panel__outline-button c-diversion-panel__outline-button--copy icon-link"
```

Comment:
Between these two suggestions
- https://github.com/mbta/skate/pull/2852/files#r1806786402
- https://github.com/mbta/skate/pull/2852/files#r1806794925

I don't think most of the classes are needed?
```suggestion
      className="icon-link"
```